pelican-gfm/gfm.py

```diff
@@ -30,8 +29,8 @@
 import pelican.signals
 import pelican.readers
 
-from . import gfmSetup
-from . import Settings
+import gfmSetup
+import Settings
```

Comment:
Can you show *what*'s not working with:

```
from . import gfmSetup
```

That should work.

Reply:
I'm sorry, I was making notes for myself there. I couldn't get the unittest to work and still have test content be generated. that seems to be corrected now.